data/wordlists.yml

```diff
@@ -10550,13 +10550,13 @@ uplay:
   categories:
   - misc
 
-13:
+thirteen:
```

Comment:
Too generic. Needs to indicate where the wordlist is from and what does it contain.